src/index.d.ts

```diff
@@ -1305,6 +1305,76 @@ export interface SearchBarBase extends TextInputProperties {
   onChangeText?(text: string): void;
 }
 
+export class Tooltip extends React.Component<TooltipProps, any> {}
+
+export interface TooltipProps {
+  /**
+   * sets backgroundColor of the tooltip and pointer.
+   */
+  backgroundColor?: string;
+  /**
+   * Color to highlight the item the tooltip is surrounding.
+   */
+
+  highlightColor?: string;
+  /**
+   * function which gets called on closing the tooltip.
+   */
+
+  onClose?: () => any;
+  /**
+   * function which gets called on closing the tooltip.
+   */
+
+  onOpen?: () => any;
+  /**
+   * function which gets called on opening the tooltip.
+   */
+
+  pointerColor?: string;
+  /**
+   * Color of tooltip pointer, it defaults to the backgroundColor if none passed .
+   */
+
+  toggleOnPress?: Function;
+   /**
+    * Flag to determine to toggle or not the tooltip on press.
+    */
+
+  popover?: React.ReactElement<{}>;
+  /**
+   * Component to be rendered as the display container.
+   */
+
+  containerStyle?: {}
+  /**
+   * Passes style object to tooltip container
+   */
+
+  height: number;
+  /**
+   * 
+   * Tooltip container height. Necessary in order to render the container in the correct place. Pass height according to the size of the content rendered inside the container.
+   */
+
+  width: number;
+  /**
+   * Tooltip container width. Necessary in order to render the container in the correct place. Pass height according to the size of the content rendered inside the container.
+   */
+
+
+  withOverlay?: boolean;
+  /**
+   *  Flag to determine whether or not dislay overlay shadow when tooltip is open.
+   */
+
+  withPointer?: boolean;
```

Comment:
Cosmetic as well - the pattern for descriptions is:

```
comment
prop

comment
prop
```

So the new line is between the two props. The description of the prop goes just above it.